crates/core_arch/src/wasm32/simd128.rs

```diff
@@ -297,47 +297,47 @@ pub unsafe fn v128_load(m: *const v128) -> v128 {
 #[inline]
 #[cfg_attr(test, assert_instr(v128.load8x8_s))]
 #[target_feature(enable = "simd128")]
-pub unsafe fn v128_load8x8_s(m: *const i8) -> v128 {
+pub unsafe fn i16x8_load8x8(m: *const i8) -> v128 {
     transmute(simd_cast::<_, i16x8>(*(m as *const i8x8)))
 }
 
 /// Load eight 8-bit integers and zero extend each one to a 16-bit lane
 #[inline]
 #[cfg_attr(test, assert_instr(v128.load8x8_u))]
 #[target_feature(enable = "simd128")]
-pub unsafe fn v128_load8x8_u(m: *const u8) -> v128 {
+pub unsafe fn u16x8_load8x8(m: *const u8) -> v128 {
```

Comment:
Slight nit: I really feel that these intrinsics need an `ext` *somewhere* in the name to indicate what is going on. Perhaps `u16x8_loadext8x8`?

Reply:
That seems reasonable to me, yeah. There's sort of two conventions in wasm simd, one with `extadd` and `extmul` but another with `extend`. Perhaps `u16x8_load8x8_extend`?

I don't really mind one way or another, I just figured it was better to use the `u16x8` name here instead of `v128` to represent the output type lanes.

Reply:
I'd keep the naming close to `i16x8_extend_low_i8x16` since the behavior is very similar:
- The destination type is always `i`. The result is always a valid signed integer even if we zero-extended from an unsigned value.
- The source type is either `i8x8` or `u8x8` depending on whether we are sign or zero extending.

So perhaps something like `i16x8_load_extend_i8x8`/`i16x8_load_extend_u8x8`?